pkg/sql/pg_catalog.go

```diff
@@ -2775,6 +2774,53 @@ CREATE TABLE pg_catalog.pg_type (
 				})
 			})
 	},
+	indexes: []virtualIndex{
+		{
+			partial: false,
+			populate: func(ctx context.Context, constraint tree.Datum, p *planner, db *sqlbase.ImmutableDatabaseDescriptor,
+				addRow func(...tree.Datum) error) (bool, error) {
+
+				h := makeOidHasher()
+				nspOid := h.NamespaceOid(db, pgCatalogName)
+				coid := tree.MustBeDOid(constraint)
+				ooid := oid.Oid(int(coid.DInt))
+
+				// Check if it is a predefined type
```

Comment:
[nit] period at the end of this sentence

Reply:
Done